internal/provider/data_source_packer_image.go

```diff
@@ -84,6 +84,11 @@ func dataSourcePackerImage() *schema.Resource {
 				Type:        schema.TypeMap,
 				Computed:    true,
 			},
+			"revoke_at": {
+				Description: "The revocation time of this build",
```

Comment:
What is the zero value for this field when there is no revocation time? 

I think it could be helpful to indicate what format the output would be in and what it is set to if no revocation time is available.
```
				Description: "The revocation time of this build. This field will be null for any build that has not been revoked or scheduled for revocation. ",
```